cmd/glasskube/cmd/list.go

```diff
@@ -19,6 +20,7 @@ type ListCmdOptions struct {
 	ShowDescription   bool
 	ShowLatestVersion bool
 	More              bool
+	ListFormat        string
```

Comment:
Cobra (the CLI library we use) supports custom types for flags by adding `String()` `Set()` and `Type()` functions to a type. I think it would make sense to use this feature here, since the same flag will be added to multiple commands in the future. 

We do something similar with the bootstrap type, which you can look at for reference: https://github.com/glasskube/glasskube/blob/main/pkg/bootstrap/type.go

Let me know if you have any questions regarding this.

Reply:
Well here in the `init` function should have the `VarP` instead of `StringVarP`
Because I took the reference to the Bootstrap Type command and it has `VarP`
https://github.com/glasskube/glasskube/blob/c2d0fcf4306aa9c9f5c716143d644004e6a5f646/cmd/glasskube/cmd/bootstrap.go#L87

Reply:
That's right. 

First, you add a custom string type. 
```go
type OutputFormat string
```
Then you define possible values.

```go
var (
  OutputFormatJSON OutputFormat = "json"
  OutputFormatYAML OutputFormat = "yaml"
)
```

Next, you add the required functions as mentioned and change the option type and flag declaration as you described.